scripts/analyze.sh

```diff
@@ -189,27 +200,32 @@ if check_for_require arduino101_pins || check_config_file ZJS_ARDUINO101_PINS; t
 fi
 
 interval=$(grep setInterval $SCRIPT)
-if [ $? -eq 0 ]; then
+if [ $? -eq 0 ] || check_config_file ZJS_TIMERS; then
     MODULES+=" -DBUILD_MODULE_TIMER"
+    echo "export ZJS_TIMERS=y" >> zjs.conf.tmp
 else
     timeout=$(grep setTimeout $SCRIPT)
     if [ $? -eq 0 ]; then
         MODULES+=" -DBUILD_MODULE_TIMER"
+        echo "export ZJS_TIMERS=y" >> zjs.conf.tmp
     else
         immediate=$(grep setImmediate $SCRIPT)
         if [ $? -eq 0 ]; then
             MODULES+=" -DBUILD_MODULE_TIMER"
+            echo "export ZJS_TIMERS=y" >> zjs.conf.tmp
```

Comment:
These two lines are repeated 3x, I think they could be here once if you restructured the if statements to check the reverse like if !0 && ! check_config && !timeout && !immediate, then timers=false. Default, timers=true. But not a big deal.